### PR TITLE
Base articles

### PR DESCRIPTION
**Pull request by sweetsweetstefan**
Base articles to merge in

### CODE REVIEW

File: en/Single-Entrypoint/Single-Entrypoint.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ---
-name: Single Entrypoint
+title: Single Entrypoint
```
Comment (evanluc) [inline en/Single-Entrypoint/Single-Entrypoint.md]: for category metadata we use name, you did this correctly in basic-concepts

Comment (evanluc) [inline en/Single-Entrypoint/Single-Entrypoint.md]:
```suggestion
name: Single Entrypoint
```

**Review by evanluc — APPROVED** [review]

**Comment by evanluc** [comment]
LGTM other than the little nit

File: en/Single-Entrypoint/Single-Entrypoint.md (hunk 1 above)
Comment (sweetsweetstefan) [reply en/Single-Entrypoint/Single-Entrypoint.md]: Oh lame, my bad, I have it in my notes as "Name changing to Title."  I'll change it back now.

Comment (sweetsweetstefan) [reply en/Single-Entrypoint/Single-Entrypoint.md]: Fixed

**Pull request merged**